harvest/cli.py

```diff
@@ -265,11 +267,26 @@ def _get_streamer(streamer):
     return streamer_cls()
 
 
-def _get_broker(broker):
+def _get_broker(broker, streamer, streamer_cls):
     """
-    Returns the storage instance specified by the user.
+    Returns the broker instance specified by the user.
     :broker: The type of broker to be instantiated.
     """
+
+    if broker == streamer:
+        debugger.info(f"Using streamer instance as broker. Streamer: {streamer}")
+        return streamer_cls
+
+    if broker == "streamer":
+        if streamer in brokers.keys():
+            debugger.info(f"ℹ Using streamer instance as broker. Streamer: {streamer}")
+            return streamer_cls
+        else:
+            debugger.warning(
+                f"⚠ Streamer {streamer} does not support broker methods! Using paper broker."
+            )
+            return brokers.get("paper")()
```

Comment:
If the broker is the default, then use the streamer. If the streamer is not a broker, use the paper broker.